### DISCUSSION TITLE
Why these start from NHANES 2005-2006?

### DISCUSSION THREAD

**Discussion by 41meng** [jamesjiadazhan/dietaryindex#211] [General]
These dietary indexes were calculated from NHANES 2005-2006. Why not NHANES 1999-2000? The dietary data from NHANES made changes from 1999.

**Comment by jamesjiadazhan** [comment]
Thanks for this question. The NHANES 1999-2005 functions are still under development. Once they are finished, they will be published. The next update will happen in the next 1-2 months.

**Comment by jamesjiadazhan** [comment]
I just added the DII and HEI2015 for 1999-2000 data. The function's name is DII_NHANES_MPED. Good luck.

Let me know if you have other questions. If possible, please give our work a Star on GitHub and cite our work when you publish the work. It helps recognize and appreciate our intellectual efforts and contributions.

<img width="594" alt="image" src="https://github.com/jamesjiadazhan/dietaryindex/assets/108076575/a657d880-02a0-4af6-b08a-810e643d36a2">

Jiada James Zhan, Rebecca A Hodge, Anne Dunlop, et al. Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies. bioRxiv. Published online August 07, 2023:2023.08.07.548466. doi:10.1101/2023.08.07.548466

**Reply by 41meng** [reply]
Thanks for your selfless contributions. I have given your work a Star and will cite your work when I publish related work. Thanks for your work again.
